lib/main.js

```diff
@@ -10,6 +10,16 @@ const LineEndingRegExp = /\r\n|\n/g
 
 let disposables = null
 
+export let config = {
+  defaultNewFileLineEnding: {
```

Comment:
Sure, was to be explicit about the fact that it was just applied for new files.

Will update PR.